src/views/SignDidExtrinsic/SignDidExtrinsic.tsx

```diff
@@ -250,45 +257,46 @@ export function SignDidExtrinsic({ identity }: Props) {
 
       const { seed } = await passwordField.get(event);
 
-      const keyRelationship = Did.getKeyRelationshipForTx(extrinsic);
+      const keyRelationship = getVerificationRelationshipForTx(extrinsic);
 
       if (!keyRelationship) {
         throw new Error('No key relationship found');
       }
 
       if (keyRelationship === 'assertionMethod') {
         const api = ConfigService.get('api');
-        const attestationKey = deriveAttestationKeyFromSeed(seed);
+        const keypair = deriveAttestationKeyFromSeed(seed);
+        const attestationKey = verificationMethodFromKeypair(keypair, did);
+
+        fullDidDocument.verificationMethod?.push(attestationKey);
```

Comment:
Let’s put a comment here about why we’re certain this array exists

Reply:
Now I'm wondering if I'm certain it exists 😅 It must exist because all DIDs have at least 1 verification method unless they're deactivated, right?

Reply:
Yes, and that would be the authentication key.

Reply:
I also thought that, but scanning through code should not cause such mental exercises, so a comment should say this :)